File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/DataMessage.java
package info.nightscout.androidaps.plugins.pump.insight.satl;

import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf;

public class DataMessage extends SatlMessage {

    private ByteBuf data;

    @Override
    protected void parse(ByteBuf byteBuf) {
        data = byteBuf;
    }

    public ByteBuf getData() {
        return this.data;
    }

    public void setData(ByteBuf data) {
        this.data = data;
    }
}

File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/DataMessage.kt
package info.nightscout.androidaps.plugins.pump.insight.satl

import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf

class DataMessage : SatlMessage() {

    public override var data: ByteBuf? = null
    override fun parse(byteBuf: ByteBuf?) {
        data = byteBuf
    }
}